docs/source/using/experimental-features.adoc

```diff
@@ -0,0 +1,89 @@
+include::variables.adoc[]
+
+= Experimental Features
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 2
+
+toc::[]
+
+[[experimental-features-overview]]
+== Overview
+
+If you want to get early access to some upcoming features and experiment, you can set the environment variable `MINISHIFT_ENABLE_EXPERIMENTAL`, which makes additional features flags available:
+
+----
+$ export MINISHIFT_ENABLE_EXPERIMENTAL=y
+----
+
+[IMPORTANT]
+====
+Experimental features are not officially supported, and might break or result in unexpected behavior.
+To share your feedback on these features, you are welcome to link:https://github.com/minishift/minishift#community[contact the {project} community].
+====
+
+[[enabling-experimental-oc-flags]]
+== Enabling experimental `oc cluster up` flags
+
+By default, {project} does not expose all link:https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md[`oc cluster up`] flags in the {project} CLI.
+
+You can set the `MINISHIFT_ENABLE_EXPERIMENTAL` environment variable to enable the following options for the xref:../command-ref/minishift_start.adoc#[`minishift start`] command:
+
+`service-catalog`::
+Enables provisioning the OpenShift link:https://docs.openshift.org/latest/architecture/service_catalog/index.html[service catalog].
+
+`extra-clusterup-flags`::
+Enables passing flags that are not directly exposed in the {project} CLI directly to `oc cluster up`.
+
+[[hyperv-static-ip]]
+== Assign IP address to Hyper-V (experimental)
+
+Since the Internal Virtual Switch for Hyper-V does not provide a DHCP offer option, an IP address needs to be provided in a different way.
+For Hyper-V a functionality is provided to assign an IP address on startup using the Data Exchange Service.
+
+[IMPORTANT]
+====
+-  While the default image is B2D, this only works with the CentOS/RHEL based image in combination with Hyper-V. The B2D image experiences
+   a problem when the values are being send to the {project} instance and consumed by the B2D iso. We are looking into the issue and hope
+   to provide a solution in the coming future.
+====
+
+To make this work you need to create a Virtual Switch using NAT
```

Comment:
The lack of ending punctuation and use of a single-item list here feels awkward. Can we incorporate the link directly into this sentence, remove the list, and add the appropriate punctuation?

Reply:
+1